### PR TITLE
Change the Inner pages to align with the home page

### PR DESCRIPTION
**Pull request by daneshk**

### CODE REVIEW

File: src/components/templates/Layout.tsx
Comment (ManujaDewmina) [inline src/components/templates/Layout.tsx:1]: Screenshot from 2023-09-10 20-59-03
Top bar icons are not in corners in large screen view

**Review by ManujaDewmina — COMMENTED** [review]
Top bar icons are not in corners in large screen view
Screenshot from 2023-09-10 20-59-03

**Comment by daneshk** [comment]
> Top bar icons are not in corners in large screen view Screenshot from 2023-09-10 20-59-03

yes, I'm not sure how to fix it though. I will check

**Comment by daneshk** [comment]
I merged the PR because we need these changes to continue with other work. I will address the fixes in separate PR

**Pull request merged**